src/main/java/it/pagopa/ecommerce/commons/repositories/PaymentRequestInfo.java

```diff
@@ -40,7 +40,8 @@ public record PaymentRequestInfo(
         @Nullable String activationDate,
         @Nullable IdempotencyKey idempotencyKey,
         @Nullable List<PaymentTransferInfo> transferList,
-        @Nullable Boolean isAllCCP
+        @Nullable Boolean isAllCCP,
+        @Nullable String creditorReferenceId
```

Comment:
fix javadoc adding `creditorReferenceId` parameter to the below persistence constructor